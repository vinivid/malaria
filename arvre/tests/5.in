i Lea
i Guta
i Ze
i Ze
