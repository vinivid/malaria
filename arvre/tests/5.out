digraph {
"0 Lea" ->"1 Guta" 
"0 Lea" ->"2 Ze" 
}

digraph {
"0 Lea" ->"1 Guta" 
"0 Lea" ->"2 Ze" 
}

digraph {
"0 Lea" ->"1 Guta" 
"0 Lea" ->"2 Ze" 
}

